pom.xml

```diff
@@ -11,7 +11,8 @@
   <version>1.11.2-1-SNAPSHOT</version>
   <packaging>hpi</packaging>
   <name>Jenkins jQuery plugin</name>
-  <url>http://wiki.jenkins-ci.org/display/JENKINS/jQuery+Plugin</url>
+  <description>This plugin is a library plugin for other plugins to share common jQuery. It also allows users to use jQuery on each view descriptions.</description>
```

Comment:
What do you think _This plugin provides an stable version of jQuery Javascript Library_?